lib/shoryuken/polling/strict_priority.rb

```diff
@@ -38,6 +38,10 @@ def active_queues
           .reverse
       end
 
+      def unpause_queue(queue)
+        @paused_until[queue] = Time.now
```

Comment:
I like the log message in `WeightedRoundRobin#unpause_queue`. It'd be good to have one here, too.